website/client/src/components/tasks/taskModal.vue

```diff
@@ -1064,25 +1092,23 @@ export default {
       dayMapping: 'constants.DAY_MAPPING',
       ATTRIBUTES: 'constants.ATTRIBUTES',
     }),
+    // region advanced settings
+    advancedSettingsShowAdjustCounter () {
+      return this.task.type === 'habit'
+                  && this.isUserTask && this.purpose === 'edit' && (this.task.up || this.task.down);
```

Comment:
that is a wild indent lol

Reply:
interesting - that these kinds are .. NOT?? formatted? - just tried and changed it and the usually "I'll reformat everything and their semicolon" just ignories that ... huh

Reply:
maybe we need to find / add a new linting/formatter rule for that (but not in this PR =D )